interfaces/Portalicious/src/app/components/registration-page-layout/registration-page-layout.component.ts

```diff
@@ -46,6 +47,8 @@ import { RegistrationActionMenuService } from '~/services/registration-action-me
 import { RegistrationLookupService } from '~/services/registration-lookup.service';
 import { TranslatableStringService } from '~/services/translatable-string.service';
 
+import { DuplicateStatus } from '../../../../../../services/121-service/src/registration/enum/duplicate-status.enum';
```

Comment:
```suggestion
import { DuplicateStatus } from '@121-service/registration/enum/duplicate-status.enum';
```

Not something like this? (I don't know which tool should fix this, `Prettier`/`ESLint`...